playbooks/technology_radar/data.csv

```diff
@@ -1,29 +1,32 @@
 name,ring,quadrant,isNew,description
-GraphQL,adopt,languages & frameworks,TRUE,
-Objective-C,adopt,languages & frameworks,FALSE,
+GraphQL,adopt,languages & frameworks,TRUE,"Appreciated for its flexibility and efficiency for clients."
+JSON,adopt,languages & frameworks,FALSE,"We prefer JSON over most other data formats for its self-describing properties."
+Objective-C,adopt,languages & frameworks,FALSE,"We generally opt for React Native, but when necessary for quality or usability, we will build native features in Objective-C using UIKit. Core Eigen infrastructure and React Native itself is written in Objective-C."
 Rails,adopt,languages & frameworks,FALSE,
-React Native,adopt,languages & frameworks,FALSE,
-React Native,adopt,languages & frameworks,FALSE,
+React Native,adopt,languages & frameworks,FALSE,"New iOS work should use React Native. See also Objective-C."
 React.js,adopt,languages & frameworks,FALSE,
 Relay,adopt,languages & frameworks,FALSE,
 Ruby,adopt,languages & frameworks,FALSE,
+Spark,adopt,languages & frameworks,FALSE,"Artsy manages a Spark cluster and patterns exist to load data from the main Gravity database as well as other systems, distribute processing among all of the available nodes, and interact with S3, HDFS, or Redshift. See the Cinder repo."
 SQL,adopt,languages & frameworks,FALSE,
 styled-components,adopt,languages & frameworks,FALSE,
 Typescript,adopt,languages & frameworks,FALSE,
-UIKit,adopt,languages & frameworks,FALSE,
+UIKit,adopt,languages & frameworks,FALSE,"See Objective-C."
 AWS,adopt,platforms,FALSE,
 Docker,adopt,platforms,FALSE,
 HSTS,adopt,platforms,FALSE,
-Kubernetes,adopt,platforms,FALSE,
+Kubernetes,adopt,platforms,FALSE,"We manage staging and production clusters, and have solutions for logging, monitoring, alerting, scaling, and routing. See Hokusai and Substance repos."
 Redis,adopt,platforms,FALSE,
+Responsive web pages,adopt,techniques,FALSE,"Rather than mobile- or desktop-specific designs, we prefer pages to degrade gracefully."
```

Comment:
Bit related to an above comment, but thoughts on adding an entry for the Opaque Ids technique?